test/stepDefinitions/create-caseSteps.js

```diff
@@ -229,4 +237,23 @@ defineSupportCode(function ({ Given, When, Then}) {
     await baseSteps.fillOutAndSubmitForm();
   });
 
+  Given(/^I have created a case with fixed list item$/, async function() {
+    await baseSteps.navigateToCreateCasePage();
+    await caseWizardPage.interactWithField("text");
+    await caseWizardPage.interactWithField("fixed-list", "Marriage");
+    await caseWizardPage.clickContinueButton();
+    await caseWizardPage.clickSubmitCaseButton();
+  });
+
+  When(/^The fixed list item is hidden on the first page$/, async function() {
+    let fieldPresent = await caseWizardPage.isFieldPresent('fixed-list')
+    expect(fieldPresent).to.be.false;
+  });
+
+  When(/^I continue to the 3rd page$/, async function() {
```

Comment:
mismatch between step description and implementation. We either change the implementation to jump to the page number indicated in the description, or we change the description to be: `I move forward 2 pages`